### ISSUE TITLE
What am I doing wrong or what should i adjust to be able to mine ?

### ISSUE THREAD

**Issue by dummie95** [trexminer/T-Rex#162]
![t-rex_not_starting](https://user-images.githubusercontent.com/78121130/106071476-ae7c1e80-6106-11eb-9339-460d82424c71.png)

**Comment by dummie95** [comment]
PS: I also tried cuda 11.x , same issue same warnings...

**Comment by cyprus00** [comment]
check to see if there are any spaces in your address / wallet id. any extra characters can cause issues with connection

**Comment by dummie95** [comment]
> 
> 
> check to see if there are any spaces in your address / wallet id. any extra characters can cause issues with connection

![t-rex_not_starting_2](https://user-images.githubusercontent.com/78121130/106186652-dae67800-61a4-11eb-82f3-d568e42c166c.png)

checked everything, no double spaces and and using exodus wallet address ( Ethereum - receive address)

**Comment by cyprus00** [comment]
your graphics card is older, you may not be using CUDA 11. notice trex can't recognize your fan of the card. try older CUDA versions. download gpuz to test card stats

**Comment by dummie95** [comment]
so I try the other 2 options avaible for me and still got same results

cuda 9:
![t-rex_not_starting_3](https://user-images.githubusercontent.com/78121130/106191281-f2c0fa80-61aa-11eb-9989-0ab1301f659c.png)

cuda 11:
![t-rex_not_starting_4](https://user-images.githubusercontent.com/78121130/106191335-08362480-61ab-11eb-812d-74d5b7bfc179.png)

**Comment by cyprus00** [comment]
check your firewall and windows defender settings for trex.exe or related programs

**Comment by cyprus00** [comment]
https://developer.nvidia.com/cuda-downloads try a fresh install of cuda and your gpu drivers. make a new copy of your .bat launching file, double check all your information.

**Comment by dummie95** [comment]
did it all, 2x fresh installs etc.. still no progress. May be I should change something in my nvidia control panel ? I mean it is optimized for games right now but thats it.

**Comment by cyprus00** [comment]
do you have msi afterburner installed? it's used for optimizing your mining / power performance. check your pci connections on your mobo and the power cables. It's more likely that the miner is not getting network access, try ethernet or a different network.

here's a similar issue someone had: https://github.com/nanopool/Claymore-Dual-Miner/issues/101

try lowering your fan speeds in afterburner for your card and running the miner again. mine are set to 50%

**Comment by dummie95** [comment]
I had the software already downloaded (afterburner) but I have no access to controls of the fanspeed. I tried to gain access to them by pressing "auto" which is above the fan speed slider and even the settings and it didnt help too.
I also have no acces to "core voltage" and "shader clock" sliders, although I did unlock and switched them on in the settings tab.
Now I'm wondering if all this missery just because I'm trying to do this mining with the laptop or does this has nothing to do with it ?
Laptop is pretty new, max 5 months old and I do hear fans working on this laptop but these may be general fans which work faster when laptop is under heavy load. Is there any command that I could use in .bat file to disable fan speed check within the mining software ?

**Comment by dummie95** [comment]
specs: i7 - 9750H (Hexa core (6))
          16gb ram
          gtx 1650 (4gb)
          win10 pro

**Comment by cyprus00** [comment]
check if your profiles are locked on the right of the ui. disable auto fan and you can set the slider manually. core volage is locked automatically; you need to grant yourself access through settings on the program. read through the config file and make sure your data is all formatted properly.

in all honesty, you aren't able to mine on a 4gb card (since december of last year). there are some workarounds though. see below.

https://www.nicehash.com/blog/post/how-to-keep-mining-on-4gb-graphics-cards#!

**Comment by trexminer** [comment]
Are you trying to mine ETH or ETC? From the first screenshot you seem to be trying to connect to ETC stratum but specify `ethash` as the algorithm (for ETH).

**Comment by dummie95** [comment]
yes, I made that mistake but corrected it by going back to eth.bat file. This also not reallly fixed my issue, but I do agree it was a mistake made by me.

**Comment by dummie95** [comment]
actually I tried so many mining softwares right now, I think 5 or 7 and none of them work on this machine so probably it has something to do with this laptop. I did fully disable my antivirus and windows firewall (included in antivirus) but nothing seem to work. Now I'm trying to go for nicehash and just try that out

**Comment by dummie95** [comment]
I'm always open for any new suggestions, I really appreciate that. Thanks in advance 👍 👍

**Comment by trexminer** [comment]
Try running a few bat files from the miner archive as is. If all of them fail with network connection error, then it's definitely something network / firewall related.

**Comment by dummie95** [comment]
I do want to say it works really heavy , i mean my gpu usage is 100% and temperature max 69°C.
The problem is with this device I don't have access to fanspeed controls (msi afterburner). Tried all to unlock it (settings as well as the "auto" button above fanspeedslider) but no result, so I think its just not supported here.
Now I can only control the "Core Clock" & "Memory Clock" sliders and thats it. So my question is which one should I lower of these 2 sliders to limit gpu usage to 90% for example ?

**Comment by trexminer** [comment]
> So my question is which one should I lower of these 2 sliders to limit gpu usage to 90% for example ?
Try reducing the power limit
